netci.groovy

```diff
@@ -813,7 +818,7 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
 
     // Check scenario.
     switch (scenario) {
-        case 'default':
+        case ['default', 'innerloop']:
```

Comment:
It seems like if we are going to trigger pri-1 (`default`) jobs on push, and those are always a superset of `innerloop`, that we should never have `innerloop` push jobs. Same for cron jobs: no need for any innerloop cron jobs if the equivalent, superset, pri-1 job is there.

Reply:
:+1: